tests/functional/heketi/test_heketi_brick_evict.py

```diff
@@ -111,19 +113,24 @@ def _wait_for_gluster_pod_after_node_reboot(self, node_hostname):
                 self.ocp_client, gluster_pod, service, "active", state)
 
     @pytest.mark.tier4
-    def test_brick_evict_with_node_down(self):
+    @ddt.data(3, "")
```

Comment:
It looks like these two tests has completely different steps. I would suggest to add different tests instead of adding ddt as many steps are controlled using if conditions.

Reply:
Removed the DDT can you please check?